core/types.go

```diff
@@ -0,0 +1,27 @@
+package core
+
+import (
+	"math/big"
+
+	"github.com/ChainSafe/gossamer/common"
+)
+
+// Block defines a state block
+type Block struct {
+	Header BlockHeader
+	Body   BlockBody
+}
+
+// BlockHeader is a state block header
+type BlockHeader struct {
+	ParentHash     common.Hash
+	Number         *big.Int
+	StateRoot      common.Hash
+	ExtrinsicsRoot common.Hash
+	Digest         []byte
+	// TODO: Not part of spec, can potentially remove
+	Hash common.Hash
+}
+
+// BlockBody is the extrinsics inside a state block
+type BlockBody struct{}
```

Comment:
could you change this to be []byte? in Definition 3.8 the block body is a scale encoding of all the extrinsics, which is a byte array

Reply:
Done